overlord/configstate/configcore/runwithstate.go

```diff
@@ -66,32 +66,51 @@ func init() {
 	addWithStateHandler(validateNetplanSettings, handleNetplanConfiguration, &flags{coreOnlyConfig: true})
 }
 
-// Conf is an interface describing both state and transaction.
-type Conf interface {
+// RunTransaction is an interface describing both state and transaction.
```

Comment:
we can maybe can improve on this doc comment now?

RunTransaction is how Run access the config transaction and state.

Reply:
Done